static_obs_location: [
                  [0.15,4.6], 
                  [0.55, 5.5],
                  [0.2, 1.5],
                  [0.85,0.85],
                  [1.6, 0.15],
                  [1.8, 0.15],
                  [2.5, 0.45],
                  [4.0, 0.15],
                  [4.2, 0.15],
                  [4.4, 0.15],
                  [5.2, 0.6],
                  [5.9, 1.3],
                  [5.6, 2],
                  [5.9, 2.7],
                  [5.9, 4.1],
                  [5.9, 4.3],
                  [5.6, 5.0],
                  [5.5, 5.2],
                  [1.5, 0.83],
                  [1.16, 1.63]
                  ]